packages/edit-site/src/store/selectors.js

```diff
@@ -47,29 +47,33 @@ export const getCanUserCreateMedia = createRegistrySelector( ( select ) => () =>
 /**
  * Returns the settings, taking into account active features and permissions.
  *
- * @param {Object} state Global application state.
+ * @param {Object}   state             Global application state.
+ * @param {Function} setIsInserterOpen Setter for the open state of the global inserter.
  *
  * @return {Object} Settings.
  */
 export const getSettings = createSelector(
-	( state ) => {
+	( state, setIsInserterOpen ) => {
```

Comment:
One thought! For the second arg, would it be helpful if it were an object? Like:

`createSelector( state, options = {})`

My thinking is that it would more gracefully handle additional enhancements to `getSettings` as Edit Site evolves.

(Totally not a blocker though!)

Reply:
Yes!, but since it's not public yet, let's just change it when the need arises.